diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index d50ffdfbcce34..7ca2a1a3f6f86 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -288,6 +288,9 @@ source_set("extensions") {
     "external_pref_loader.h",
     "external_provider_impl.cc",
     "external_provider_impl.h",
+    "browseros_extension_constants.h",
+    "browseros_external_loader.cc",
+    "browseros_external_loader.h",
     "external_provider_manager.cc",
     "external_provider_manager.h",
     "external_provider_manager_factory.cc",
